force-app/main/default/classes/AddingContactToAccountUtility.cls

```diff
@@ -0,0 +1,13 @@
+public  with sharing class AddingContactToAccountUtility {
+    public static void addContact(List<Account> accounts){
+            For(Account account:accounts){
+                Contact contact=new Contact( FirstName=account.Name, LastName='testLastName',AccountId=account.Id);
+
+                System.Debug(contact.AccountId +' =? ' + account.Id);
+                insert(contact);
```

Comment:
Tutaj bardzo ważny temat, które trzeba przetstrzegać, nigdy nie robimy DML wewnątrz pętli, zawsze zbieramy liste wcześniej, tutaj dodajemy do listy i poza listą robimy insert, ponieważ sf ma limit 100 DML w tranksacji, więc wystarczy, że ktoś masowo doda 100 accountów to ta pętla zatrzyma całą transakcje. Więcej opowiemy sobie na callu